<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CUACSView</class>
 <widget class="QMainWindow" name="CUACSView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>721</width>
    <height>521</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CUACSView</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabView">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="focusPolicy">
       <enum>Qt::ClickFocus</enum>
      </property>
      <property name="toolTipDuration">
       <number>-2</number>
      </property>
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>View Animals</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QTableWidget" name="animalTbl">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
              <horstretch>10</horstretch>
              <verstretch>10</verstretch>
             </sizepolicy>
            </property>
            <property name="editTriggers">
             <set>QAbstractItemView::NoEditTriggers</set>
            </property>
            <property name="columnCount">
             <number>7</number>
            </property>
            <attribute name="horizontalHeaderCascadingSectionResizes">
             <bool>false</bool>
            </attribute>
            <attribute name="horizontalHeaderDefaultSectionSize">
             <number>95</number>
            </attribute>
            <column>
             <property name="text">
              <string>Name</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Species</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Breed</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Gender</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Date of Birth</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Age(Y/M)</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Vaccinated</string>
             </property>
            </column>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="clientTab">
       <attribute name="title">
        <string>View Clients</string>
       </attribute>
       <widget class="QTableWidget" name="clientTable">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>10</y>
          <width>691</width>
          <height>431</height>
         </rect>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>10</horstretch>
          <verstretch>10</verstretch>
         </sizepolicy>
        </property>
        <property name="editTriggers">
         <set>QAbstractItemView::NoEditTriggers</set>
        </property>
        <property name="columnCount">
         <number>5</number>
        </property>
        <attribute name="horizontalHeaderCascadingSectionResizes">
         <bool>true</bool>
        </attribute>
        <attribute name="horizontalHeaderDefaultSectionSize">
         <number>137</number>
        </attribute>
        <attribute name="horizontalHeaderHighlightSections">
         <bool>true</bool>
        </attribute>
        <attribute name="horizontalHeaderStretchLastSection">
         <bool>false</bool>
        </attribute>
        <attribute name="verticalHeaderCascadingSectionResizes">
         <bool>true</bool>
        </attribute>
        <column>
         <property name="text">
          <string>Username</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>First Name</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Last Name</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Email</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Phone Number</string>
         </property>
        </column>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Add Animal</string>
       </attribute>
       <widget class="QPushButton" name="addAnimalBtn">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="geometry">
         <rect>
          <x>560</x>
          <y>150</y>
          <width>82</width>
          <height>25</height>
         </rect>
        </property>
        <property name="toolTipDuration">
         <number>-1</number>
        </property>
        <property name="text">
         <string>Add animal</string>
        </property>
        <property name="checkable">
         <bool>false</bool>
        </property>
       </widget>
       <widget class="QLabel" name="emptyAnimalLbl">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>150</y>
          <width>311</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>Please ensure every field is entered correctly</string>
        </property>
       </widget>
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>641</width>
          <height>151</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_13">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout">
             <item>
              <widget class="QLabel" name="label">
               <property name="text">
                <string>*Name:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="nameTxt"/>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <item>
              <widget class="QLabel" name="label_2">
               <property name="text">
                <string>*Breed:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="breedTxt"/>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_5">
             <item>
              <widget class="QLabel" name="label_7">
               <property name="text">
                <string>Date of Birth(yyyy-mm-dd):</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="DOBTxt">
               <property name="toolTip">
                <string/>
               </property>
               <property name="toolTipDuration">
                <number>-1</number>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <item>
              <widget class="QLabel" name="label_3">
               <property name="text">
                <string>*Age:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_4">
               <property name="text">
                <string>Years</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSpinBox" name="yearsSpn">
               <property name="prefix">
                <string/>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_5">
               <property name="text">
                <string>Months</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSpinBox" name="monthsSpn">
               <property name="maximum">
                <number>12</number>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_4">
             <item>
              <widget class="QLabel" name="label_6">
               <property name="text">
                <string>*Gender:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="genderCombo">
               <property name="currentText">
                <string/>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QCheckBox" name="vaccinatedChk">
               <property name="layoutDirection">
                <enum>Qt::RightToLeft</enum>
               </property>
               <property name="text">
                <string>Vaccinated?</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_6">
             <item>
              <widget class="QLabel" name="label_8">
               <property name="text">
                <string>*Species:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="speciesCombo">
               <property name="currentText">
                <string/>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_3">
       <attribute name="title">
        <string>Add Client</string>
       </attribute>
       <widget class="QWidget" name="horizontalLayoutWidget">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>701</width>
          <height>227</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_20">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_8">
             <item>
              <widget class="QLabel" name="fNameLbl">
               <property name="text">
                <string>*First Name:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="firstNameTxt"/>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_7">
             <item>
              <widget class="QLabel" name="lNamelbl">
               <property name="text">
                <string>*Last Name</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="lastNameTxt"/>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_17">
             <item>
              <widget class="QLabel" name="emailLbl">
               <property name="text">
                <string>*Email: </string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="emailTxt"/>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_9">
             <item>
              <widget class="QLabel" name="userLbl">
               <property name="text">
                <string>*Username: </string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="usernameTxt"/>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_11">
             <item>
              <widget class="QLabel" name="passLbl">
               <property name="text">
                <string>*Password:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="passwordTxt"/>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_10">
             <item>
              <widget class="QLabel" name="confPassLbl">
               <property name="text">
                <string>*Confirm Password:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="confirmPasswordTxt"/>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QLabel" name="passConLbl">
             <property name="text">
              <string>Please ensure both passwords match</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_5">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_15">
             <item>
              <widget class="QLabel" name="postLbl">
               <property name="text">
                <string>*Postal Code</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="postalCodeTxt"/>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_12">
             <item>
              <widget class="QLabel" name="cityLbl">
               <property name="text">
                <string>*City:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="cityTxt"/>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_14">
             <item>
              <widget class="QLabel" name="provLbl">
               <property name="text">
                <string>*Province:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="provinceCombo"/>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_16">
             <item>
              <widget class="QLabel" name="addLn1lbl">
               <property name="text">
                <string>*Address Line 1:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="addressLine1Txt"/>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_19">
             <item>
              <widget class="QLabel" name="addLn2Lbl">
               <property name="text">
                <string>Address Line 2:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="addressLine2Txt"/>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_18">
             <item>
              <widget class="QLabel" name="phoneLbl">
               <property name="text">
                <string>*Phone Number (xxx)-xxx-xxxx:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="phoneTxt"/>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QPushButton" name="addClientBtn">
             <property name="text">
              <string>Add Client</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
       <widget class="QLabel" name="emptyClientLbl">
        <property name="geometry">
         <rect>
          <x>340</x>
          <y>230</y>
          <width>231</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>Items marked with a * are required</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_9">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>230</y>
          <width>291</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>Passwords can include letters and numbers</string>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
